README.md

```diff
@@ -18,8 +18,10 @@ Pending
 
 To get a local copy up and running follow these simple example steps.
 
-- Download the project
-- Run it in a live Server
+- clone this repo
+- run `npm install` to install 
+- run `npm run build` to create the production
+- enter `/dist` file and open index.html
```

Comment:
- [ ] I will recommend you to be more specific with these bullet points, explain it as if someone with no coding experience is reading this READMe file:
    - How do I clone the repo? 🤔 
    - You are using webpack, so you don't need to enter the dist file to run the project, you can just run the live server with `npm run start` 😄

Reply:
Hi @helman101 ... I've made the change but I'm unable to send for another review because I'm out of the times I send the review..
* Another thing, about your feedback, cloning a repo is so easy that someone with little to no experience can still be able to find a way to clone the repo.
* Additionally, I find that if you're on github, you have to have some coding experience or at least cloning a repo.